flixel/tweens/FlxTween.hx

```diff
@@ -735,3 +853,15 @@ class FlxTweenManager extends FlxBasic
 		}
 	}
 }
+
+class ThenCommand
```

Comment:
Not sure I like the name `ThenCommand`. `ChainedTween` maybe? That doesn't really cover the wait use-case though.. `ChainedCommand`? `ChainedAction`?

Thoughts?